config/config.yaml

```diff
@@ -174,15 +174,6 @@ branch-protection:
           branches:
             master:
               protect: true
-        falcosidekick-ui:
-          required_status_checks:
-            contexts:
-              - "ci/circleci: lint"
-              - "ci/circleci: test"
-              - "ci/circleci: build-image"
```

Comment:
Please, only remove `required_status_checks` block. We still need the rest of the configuration. :pray: